tests/test_basic.py

```diff
@@ -261,17 +261,15 @@ def test():
 
         with self.assertRaises(tiledb_cloud_error.TileDBCloudError):
 
-            def test():
+            def test(data):
```

Comment:
Since this function is a duplicate of the one above, I would delete this one and add the parameter to the one above (you can call it `_`, since it’s unused).